Sources/DistributedActors/Cluster/ClusterEventStream.swift

```diff
@@ -123,7 +123,6 @@ public struct ClusterEventStream: AsyncSequence {
     }
 }
 
-// FIXME(distributed): the only reason this actor is distributed is because of LifecycleWatch
```

Comment:
This specific actor doesn't need to be DA either and with that we can probably remove `whenLocal` usages too...

Reply:
That'd need a way to implement lifecycle watch without DA... we don't have that.

Reply:
I'll make a ticket for non-DA lifecycle watching though hm hm

Reply:
https://github.com/apple/swift-distributed-actors/issues/1036